cachito/workers/pkg_managers/rubygems.py

```diff
@@ -86,7 +89,8 @@ def parse_gemlock(source_dir, gemlock_path):
             continue
         _validate_gem_metadata(gem, source_dir, gemlock_path.parent)
         source = gem.remote if gem.type != "PATH" else gem.path
-        dependencies.append(GemMetadata(gem.name, gem.version, gem.type, source))
+        branch = gem.branch if gem.branch is not None else ""
```

Comment:
Might as well use `None` instead of converting to `""`

You would just need to change the type annotation to `Optional[str]`

Reply:
Yeah, I wanted to use `None`, but I didn't know about the `Optional` annotation